src/ProjectNeon/Assets/Features/Battle/Cards/ConfirmCardSelection.cs

```diff
@@ -3,8 +3,8 @@
 public class ConfirmCardSelection : MonoBehaviour
 {
     [SerializeField] private CardPlayZone playArea;
-    [SerializeField] private GameEvent onConfirmation;
     [SerializeField] private GameObject confirmUi;
+    public GameEvent onConfirmation;
```

Comment:
Scripts should not expose their variables.

Every listener to an Event can subscribe separately.